index.rb

```diff
@@ -0,0 +1,10 @@
+require './person'
+require './capitalize'
+require './trimmer'
+
+person = Person.new(22, 'maximilianus')
+puts person.correct_name
+puts capitalized = CapitalizeDecorator.new(person)
+puts capitalized.correct_name
+puts capitalized_trimmed = TrimmerDecorator.new(capitalized)
```

Comment:
- [ ] **[OPTIONAL]** What you have is a `Trimmer` class, not `TrimmerDecorator`.

  ![pola-trim](https://user-images.githubusercontent.com/36542986/159520659-887cd147-1e04-4987-907c-689063adedbc.jpg)